modules/xml-rpc/class-xml-rpc.php

```diff
@@ -58,6 +58,11 @@ public static function disable_xml_rpc() {
 
 		// TODO: Figure how to disable XML-RPC without returning a value.
 		add_filter('wp_xmlrpc_server_class', function ( $server_class ) {
+			/**
+			 * Filters the HTTP status code returned when XML-RPC is disabled.
+			 */
+			status_header( apply_filters( 'vip_security_boost_xml_rpc_disabled_status_code', 403 ) );
```

Comment:
@raamdev nitpicking: we're not checking that the `vip_security_boost_xml_rpc_disabled_status_code` output is a valid integer number. I don't think customers will misuse this but I also believe it would be nice to ensure that, if they do, the app will continue to work properly.